### ISSUE TITLE
Transformer should support multiple independent entry points

### ISSUE THREAD

**Issue by jakemac53** [angular/angular#2919]
tl;dr; Once https://github.com/angular/angular/issues/2304 is resolved we should remove `reflection_entry_points` from the transformer config.

Today, it looks like each reflection entry point will end up importing the ng_deps file for every entry point and attempt to initialize all of them. Since there are just two independent lists in the transformer config, you can't tell which entry point corresponds to which reflection entry point.

However, once https://github.com/angular/angular/issues/2304 is implemented there should no longer be a need for the reflection_entry_points option at all, so we can just remove it.

**Comment by jakemac53** [comment]
Removing milestone and unassigning myself for now, as I will not be able to get to this before I go on vacation next week.

**Comment by jakemac53** [comment]
blocked on https://github.com/angular/angular/issues/2304

**Comment by mhevery** [comment]
This is obsolete, and no longer relevant / actionable. To keep our issues clean, we are aggressively closing them.
